services/spar/src/Spar/Types.hs

```diff
@@ -44,7 +45,18 @@ import qualified SAML2.WebSSO as SAML
 
 data Void
 
-type BindCookie = SimpleSetCookie "zbind"
+type SetBindCookie = SimpleSetCookie "zbind"
```

Comment:
Our headers are called `Z-Connection`, `Z-User` and so on; maybe for the cookie it also makes sense to call it `Z-Bind`? (And at least according to Google, cookies are case-sensitive.)

Reply:
this is not a header name, but a cookie name: https://github.com/wireapp/wire-server/blob/0285578539abae98c122c63558b43bbdd8227e45/services/brig/src/Brig/User/Auth/Cookie.hs#L191

cookie names are keys in the key-value pair list that comes in the http header `Cookie`.